pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -636,7 +636,8 @@ stdenv.mkDerivation ({
 
   };
 
-  meta = { inherit homepage license platforms; }
+  meta = {
+    broken = true; inherit homepage license platforms; }
```

Comment:
This file is the builder for Haskell packages defined in nixpkgs (similar to `stdenv.mkDerivation` for normal packages).

It is probably a mistake to set `broken = true` here.

I imagine what happened is that there is some individual Haskell package that is not working and needs to be set as broken, but instead some sort of script just marked the general builder as broken instead.

Individual Haskell packages can be marked as broken in the file `pkgs/development/haskell-modules/hackage-packages.nix` on the release branches.  (Search for the string `broken` in this file to see what this looks like.)

Reply:
yea, i noticed this, i forced pushed a change to the script im using

Reply:
thanks for noticing though!